### ISSUE TITLE
--tor command line switch opens duplicate instances

### ISSUE THREAD

**Issue by benfyvie** [brave/brave-browser#22162]
## Description 
On a Windows PC when using the --tor command line switch to open brave.exe results in TWO instances of the brave browser being opened in tor browsing mode. Using the similar switch of --incognito works as expected and only opens a single instance of the brave browser.

## Steps to Reproduce
   1. modify a brave.exe shortcut so that the target includes the --tor command line switch
   2. click on the shortcut to run it

## Actual result:
Two duplicate instances of the brave browser are opened, both are in tor browsing mode

## Expected result:
Only a single instance of the brave browser should be opened, and it should be in tor browsing mode

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)
Version 1.37.111 Chromium: 100.0.4896.79 (Official Build) (64-bit)](https://brave.com/latest/)

## Version/Channel Information:
Tested only on current release

## Other Additional Information:
The --tor command line switch is still relatively new as it was only recently added as part of issue #2105 and it likely missing a test case to ensure that only a single instance is opened when the switch is used.

## Miscellaneous Information:
Windows 7 was being used on the machine that I experience the issue on, unsure if this is relevant or not to reproduce the issue.

**Comment by rebron** [comment]
@btlechowski When you get a chance, can you check if this is Windows 7 specific?

**Comment by rebron** [comment]
cc: @darkdh

**Comment by JohnDoecoder** [comment]
This is an issue on Linux (Ubuntu 21.10) as well.
Brave was installed from `https://brave-browser-apt-release.s3.brave.com/ stable main`
Console output (Don't know if it provides any information)
```
XXX@YYY:/$ brave-browser --tor
[272622:272640:0424/161929.945927:ERROR:model_type_store_service_impl.cc(39)] Failed to initialize ModelTypeStore backend: Init@components/sync/model/model_type_store_backend.cc:109: IO error: /home/XXX/.config/BraveSoftware/Brave-Browser/Default/Sync Data/LevelDB/MANIFEST-000001: Unable to create writable file (ChromeMethodBFE: 9::NewWritableFile::4)
[272656:272656:0424/161930.042337:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
[272622:272622:0424/161930.080708:ERROR:brave_new_tab_message_handler.cc(193)] Ads service is not initialized!
[272622:272622:0424/161930.119973:ERROR:brave_new_tab_message_handler.cc(193)] Ads service is not initialized!
Apr 24 16:19:30.299 [notice] Tor 0.4.6.10 running on Linux with Libevent 2.1.12-stable, OpenSSL 1.1.1n, Zlib 1.2.12, Liblzma N/A, Libzstd N/A and Glibc 2.31 as libc.
Apr 24 16:19:30.299 [notice] Tor can't help you if you use it wrong! Learn how to be safe at https://support.torproject.org/faq/staying-anonymous/
Apr 24 16:19:30.299 [notice] Read configuration file "/home/XXX/.config/BraveSoftware/Brave-Browser/biahpgbdmdkfgndcmfiipgcebobojjkp/1.0.25/tor-torrc".
Apr 24 16:19:30.299 [notice] Read configuration file "/home/XXX/.config/BraveSoftware/Brave-Browser/biahpgbdmdkfgndcmfiipgcebobojjkp/1.0.25/tor-torrc".
Apr 24 16:19:30.301 [notice] Opening Socks listener on 127.0.0.1:0
Apr 24 16:19:30.301 [notice] Socks listener listening on port 35873.
Apr 24 16:19:30.301 [notice] Opened Socks listener connection (ready) on 127.0.0.1:35873
Apr 24 16:19:30.301 [notice] Opening Control listener on 127.0.0.1:0
Apr 24 16:19:30.302 [notice] Control listener listening on port 45209.
Apr 24 16:19:30.302 [notice] Opened Control listener connection (ready) on 127.0.0.1:45209
Apr 24 16:19:30.000 [notice] Bootstrapped 0% (starting): Starting
[272622:272622:0424/161930.526180:ERROR:CONSOLE(32)] "Unchecked runtime.lastError: Not available in Tor profile", source: chrome://newtab/ (32)
[272622:272622:0424/161930.526420:ERROR:CONSOLE(32)] "Unchecked runtime.lastError: Not available in Tor/incognito/guest profile", source: chrome://newtab/ (32)
[272622:272622:0424/161930.589399:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Not available in Tor/incognito/guest profile", source: chrome://newtab/ (0)
Apr 24 16:19:30.000 [notice] Starting with guard context "default"
[272622:272622:0424/161930.732874:ERROR:CONSOLE(32)] "Unchecked runtime.lastError: Not available in Tor profile", source: chrome://newtab/ (32)
[272622:272622:0424/161930.732931:ERROR:CONSOLE(32)] "Unchecked runtime.lastError: Not available in Tor/incognito/guest profile", source: chrome://newtab/ (32)
[272622:272622:0424/161930.753792:ERROR:CONSOLE(0)] "Unchecked runtime.lastError: Not available in Tor/incognito/guest profile", source: chrome://newtab/ (0)
Apr 24 16:19:31.000 [notice] Bootstrapped 5% (conn): Connecting to a relay
Apr 24 16:19:31.000 [notice] Bootstrapped 10% (conn_done): Connected to a relay
Apr 24 16:19:31.000 [notice] Bootstrapped 14% (handshake): Handshaking with a relay
Apr 24 16:19:31.000 [notice] Bootstrapped 15% (handshake_done): Handshake with a relay done
Apr 24 16:19:31.000 [notice] Bootstrapped 75% (enough_dirinfo): Loaded enough directory info to build circuits
Apr 24 16:19:31.000 [notice] Bootstrapped 90% (ap_handshake_done): Handshake finished with a relay to build circuits
Apr 24 16:19:31.000 [notice] Bootstrapped 95% (circuit_create): Establishing a Tor circuit
Apr 24 16:19:31.000 [notice] Bootstrapped 100% (done): Done
Apr 24 16:19:33.000 [notice] New control connection opened from 127.0.0.1.
```

**Comment by s5msung** [comment]
Having same issue, but I am on Windows 10. I added the TOR flag in the shortcut and it work. Except it opens up two private mode with TOR windows upon double clicking on the shortcut. I think it is a bug, can someone confirm. Delete my comment if I am repeating what's already been said by another user.

**"C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe" --tor** 
![image](https://user-images.githubusercontent.com/104453417/165352681-1d1773aa-f27f-4dbe-96e8-e9abb5d0e7fa.png)

my "brave://version/" is ...

<html>
<body>


Brave | 1.37.116 Chromium: 100.0.4896.127 (Official Build) (64-bit)
Revision | ff0d0695743e65305d7194f9bd309e5e1c824aa0-refs/branch-heads/4896_88@{#4}
OS | Windows 10 Version 21H1 (Build 19043.1586)
JavaScript | V8 10.0.139.15
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36


</body>
</html>

![image](https://user-images.githubusercontent.com/104453417/165353602-167fc5de-3124-4a85-956b-76ab096ef0a6.png)

**Comment by btlechowski** [comment]
#### Reproduced on Ubuntu - two Tor Windows are opened

Brave | 1.38.109 Chromium: 101.0.4951.41 (Official Build) (64-bit)
-- | --
Revision | 93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}
OS | Ubuntu 18.0 LTS

![image](https://user-images.githubusercontent.com/34715963/165782406-bcda5712-1dd9-4fd7-bd64-5d281aecf443.png)


#### Reproduced on Windows 10 - two Tor Windows are opened
![image](https://user-images.githubusercontent.com/34715963/165783067-6dc4ba07-9d52-4b38-8309-a1ed6395e363.png)

Brave | 1.38.109 Chromium: 101.0.4951.41 (Official Build) (64-bit)
-- | --
Revision | 93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}
OS | Windows 10 Version 21H2 (Build 19044.1645)

#### Reproduced on Windows 7 - two Tor Windows are opened
![image](https://user-images.githubusercontent.com/34715963/165783708-24409d8c-2e70-47d3-a9d8-d76557111cf0.png)

Brave | 1.37.114 Chromium: 100.0.4896.88 (Official Build) (64-bit)
-- | --
Revision | 4450653bfa91182e85723d8f1dee64dd6ce40ed4-refs/branch-heads/4896@{#1086}
OS | Windows 7 Service Pack 1 (Build 7601.24544)

**Comment by ccaioccesar** [comment]
Same issue here, windows 10, "C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --tor"
open two instances

**Comment by leszec** [comment]
same issue

**Comment by leszec** [comment]
same issue

https://community.brave.com/t/opening-tor-with-terminal-opens-2-tor-window/356755/4

they know it since march... cool browser ;)

**Comment by vbueb** [comment]
Same issue on ArchLinux `Version 1.40.113 Chromium: 103.0.5060.114 (Build officiel) (64 bits)`

**Comment by DannyyyOnGit** [comment]
I can also reproduce this on the latest stable version.
Version 1.41.100 Chromium: 103.0.5060.134 (Official Build) (64-bit)

Might be worth adding that it also happens with a specified url:
`-tor "https://brave.com/"`

**Comment by ToasterUwU** [comment]
I also have the same issue. 

Im trying to use brave with a modded selenium version (undetected-chromedriver) (which is officially supported from the devs of that modded version). And for getting different IPs, i wanted to use tor. 
TLDR; Im adding the argument tor to selenium to get different IPs, and i thought i was doing something wrong, but no. Its just a bug in Brave.

**Comment by fmarier** [comment]
Duplicate of https://github.com/brave/brave-browser/issues/21579
